Omise.Tests/Resources/ChargeResourceTest.cs

```diff
@@ -149,9 +151,12 @@ public async Task TestFixturesSearch()
             Assert.That(result[0].Amount, Is.EqualTo(409669));
         }
 
-        protected CreateChargeRequest BuildCreateRequest()
+        protected CreateChargeRequest BuildCreateRequest()
         {
-            return new CreateChargeRequest
+            TimeZoneInfo thailandZone = TimeZoneInfo.FindSystemTimeZoneById("Asia/Bangkok");
+            DateTime thailandTime = new DateTime(2023, 8, 9, 0, 0, 0);
+            DateTime utcTime = TimeZoneInfo.ConvertTimeToUtc(thailandTime, thailandZone);

```

Comment:
Fix the timezone across multi working environments so that tests don't fail from a machine to another (locally or on github actions server as utc is the default on server but locally depends on your timezone)